en/dev_setup/dev_env_windows_wsl.md

```diff
@@ -0,0 +1,161 @@
+# Windows WSL-Based Development Environment
```

Comment:
I have done a very rough update of this. I want to get it closer to the very simple instructions for Windows currently or for macOS. Not easy since there are lots of caveats. 

However I think we should iterate quickly, get it in, then think about next steps to make it "the way".

Reply:
Yes, I thought the same. Rather have the not yet polished way documented but not as the main way until it's further than just not documenting that it's usable at all.